ci/Jenkinsfile.macos

```diff
@@ -41,6 +41,16 @@ pipeline {
       description: 'Select app entitlements. Squish requires extra entitlements.',
       choices: ['resources/Entitlements.plist', 'resources/Entitlements_squish.plist']
     )
+    booleanParam(
+      name: 'USE_NWAKU',
+      description: 'Whether to build with nwaku or not',
+      defaultValue: params.USE_NWAKU ?: getNWakuMode(),
+    )
+    string(
+      name: 'NIMFLAGS',
+      description: 'Extra Nim flags. Examples: --verbosity:2 --passL:"-v" --passC:"-v"',
+      defaultValue: "--colors:off --nimcache:${env.WORKSPACE_TMP}/nimcache"
```

Comment:
@jrainville : nuke this part, its duplicate : https://github.com/status-im/status-desktop/pull/18690/files#diff-f605b94de757dc638bc5df043af84cec657bc17d681bd38a6547879dcee20b81R49-R53